src/Button.vala

```diff
@@ -29,6 +29,11 @@ public class PantheonCalculator.Button : Gtk.Button {
         child = lbl;
     }
 
+    public Button.from_icon_name (string icon_name) {
+        var image = new Gtk.Image.from_icon_name (icon_name);
```

Comment:
> Looks like we'll probably have to add a 24x symbolic icon for this so it's not blurry

I'm not sure if this is a correct fix, but I can fix the blurry icon by setting valign/halign of Gtk.Image to CENTER:

![image](https://github.com/user-attachments/assets/bb2a3f22-884a-4589-af2f-7cd0805c156c)

BTW actually we already have a 24px icon of `edit-clear-symbolic`: 

https://github.com/elementary/icons/blob/16896c5438925d394c72a4897ffa40fe790fc685/actions/24/edit-clear-symbolic.svg